### ISSUE TITLE
When using fp16, how to keep batch norm with fp32? 

### ISSUE THREAD

**Issue by jianlong-yuan** [open-mmlab/mmsegmentation#1507]
same as title. Like apex, it could use fp32 for all batch normlization?

**Comment by MeowZheng** [comment]
yes, please ref https://discuss.pytorch.org/t/understanding-pytorch-native-mixed-precision/98535/2
